data_safe_haven/types/enums.py

```diff
@@ -163,8 +165,8 @@ class PermittedDomains(tuple[str, ...], Enum):
 @verify(UNIQUE)
 class Ports(str, Enum):
     AZURE_MONITORING = "514"
-    CLAMAV = "11371"
     DNS = "53"
+    HKP = "11371"
```

Comment:
What does HKP stand for?

Reply:
Http Keyserver Protocol: https://en.wikipedia.org/wiki/Key_server_(cryptographic)